### DISCUSSION TITLE
Hi all!

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [smartcontractkit/full-blockchain-solidity-course-py#18] [General]
Feel free to say hello here!

**Comment by jessicaNono** [comment]
Hi

**Reply by PatrickAlphaC** [reply]
woot!

**Comment by lthibeault** [comment]
Hey Patrick, 
First off, thank you for this tutorial, I am about 10 hours into it, and have already learned a lot.
Your personal assistance with some of my issues has been extremely impressive.
I am getting an Error in the NFT section  now (deploying the advanced NFT to the development net). But I am sure I will figure it out. (not to worry I will post any issues I find in the discussion board.

I just wanted to say Thank you again, and great Job!

**Reply by lthibeault** [reply]
Went line by line by line and found I had swapped the position of the Hash and the fee in the constructor, that will team me to pay more attention to the "Order of Operation"...

Once again great tutorial!!!!! I cant wait to put some of this stuff to use and upgrade my Platform (add new features as well)

**Reply by PatrickAlphaC** [reply]
Beautiful!!! And thanks you! I'm so glad you're learning so much!

10 hours is quite the feat too! This course is no joke. You're doing so well!

Hope to see you at the upcoming Chainlink hackathon: https://chain.link/hackathon?utm_medium=team&utm_source=dev-advocate&utm_campaign=fall-2021-hackathon&utm_content=patrick

**Comment by Swiftmatic** [comment]
Peace and hello to everyone! I am a hour into the course and only slightly nervous there's not many comments here lol. My goal is to learn and gain a practical understanding from the course and connect with as many here as I can. I have a very limited understanding of html, css, java script, and boot strap. I am also pretty much new to github. Any recommendations will be very appreciated. Thanks!
